<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>901</width>
     <height>601</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>899</width>
      <height>599</height>
     </rect>
    </property>
    <widget class="QPushButton" name="addFacButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>480</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profileButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Faculty ID :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Faculty Name :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="facultyName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>G-Man Systems</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gradeButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Grades</string>
     </property>
    </widget>
    <widget class="QPushButton" name="courseButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Faculty</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FacultyID">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="facultyTable">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>471</width>
       <height>381</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Faculty ID</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Faculty Name</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="barLabel_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>901</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="othersButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Other Option</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>480</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <zorder>barLabel_2</zorder>
    <zorder>addFacButton</zorder>
    <zorder>profileButton</zorder>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>facultyName</zorder>
    <zorder>label_15</zorder>
    <zorder>gradeButton</zorder>
    <zorder>courseButton</zorder>
    <zorder>label_3</zorder>
    <zorder>FacultyID</zorder>
    <zorder>homeButton</zorder>
    <zorder>facultyTable</zorder>
    <zorder>logoutButton</zorder>
    <zorder>othersButton</zorder>
    <zorder>deleteButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
